concepts/05 Widgets/DataGrid/25 Sorting/20 API/1 Initial and Runtime Sorting.md

```diff
@@ -73,5 +133,82 @@ Change the **sortOrder** and **sortIndex** options using the [columnOption](/api
         ],
         // ...
     })
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid :ref="dataGridRefKey">
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxDataGrid } from 'devextreme-vue/data-grid';
+
+    const dataGridRefKey = "my-data-grid";
+
+    export default {
+        components: {
+            DxDataGrid
+        },
+        data() {
+            return {
+                dataGridRefKey
+            };
+        },
+        methods: {
+            sortByCountries() {
+                this.dataGrid.columnOption('Country', {
+                    sortOrder: 'desc',
+                    sortIndex: 2
+                });
+            }
+        },
+        computed: {
+            dataGrid() {
+                return this.$refs[dataGridRefKey].instance;
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DataGrid } from 'devextreme-react/data-grid';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+            this.dataGridRef = React.createRef();
+        }
+
+        get dataGrid() {
+            return this.dataGridRef.current.instance;
+        }
+
+        render() {
+            return (
+                <DataGrid ref={this.dataGridRef} ...>
+                </DataGrid>
+            );
+        }
+
+        sortByCountries() {
+            this.dataGrid.columnOption('Country', {
```

Comment:
We don't use columnOption for React, let's use React state instead
like here - https://js.devexpress.com/Documentation/Guide/Widgets/DataGrid/Filtering_and_Searching/#Header_Filter
This is more native for React